### PR TITLE
Cart management

### PR DESCRIPTION
**Pull request by Samarth-5109**
Screenshot (41)

### CODE REVIEW

**Comment by MonikaJhingan** [comment]
looks good to be merged.

**Review by MonikaJhingan — APPROVED** [review]
Looks good to be merged.

**Comment by GeetaPogul** [comment]
looks good to me.

File: pages/cart.html [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+      integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p"
+      crossorigin="anonymous"
+    />
+    <!-- <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up/2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous"/> -->
```
Comment (GeetaPogul) [inline pages/cart.html:13]: if you don't want this code then remove it

**Review by GeetaPogul — APPROVED** [review]
good to me

**Pull request merged**